app/models/post.rb

```diff
@@ -0,0 +1,8 @@
+class Post < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  has_many :comments
+  has_many :likes
+
+  validates :title, presence: true
+  validates :text, presence: true
```

Comment:
The post model must have a method that updates the posts counter for a user as it's part of the project requirement. Kindly implement that 👍